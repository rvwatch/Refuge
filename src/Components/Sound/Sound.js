import React from 'react';
import {Link} from 'react-router-dom';

export const Sound = () => {
  return (
    <section>
      <header className='excercise-nav'>
        <Link to='/mindfulness' className='prev'>meditate</Link>
        <Link to='/journal' className='next'>journal</Link>
      </header>
      <article className="video-frame">
        <iframe
          title='calming sound-track'
          width="560"
          height="315"
          // eslint-disable-next-line
          src="https://www.youtube-nocookie.com/embed/1dFvc42g8CA?rel=0&amp;showinfo=0"
          frameBorder="0"
          allow="autoplay; encrypted-media"
          allowFullScreen
        />
      </article>
    </section>
  );
};
