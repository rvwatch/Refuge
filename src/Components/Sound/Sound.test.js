import  React from 'react';
import {shallow} from 'enzyme';
import {Sound} from './Sound';

describe('Sound', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Sound />);
    expect(wrapper).toMatchSnapshot();
  });
});