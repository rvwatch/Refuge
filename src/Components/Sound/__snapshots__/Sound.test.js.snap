// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sound should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <article
        className="video-frame"
>
        <iframe
                allow="autoplay; encrypted-media"
                allowFullScreen={true}
                frameBorder="0"
                height="315"
                src="https://www.youtube-nocookie.com/embed/1dFvc42g8CA?rel=0&showinfo=0"
                width="560"
        />
</article>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <iframe
          allow="autoplay; encrypted-media"
          allowFullScreen={true}
          frameBorder="0"
          height="315"
          src="https://www.youtube-nocookie.com/embed/1dFvc42g8CA?rel=0&showinfo=0"
          width="560"
/>,
        "className": "video-frame",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "allow": "autoplay; encrypted-media",
          "allowFullScreen": true,
          "frameBorder": "0",
          "height": "315",
          "src": "https://www.youtube-nocookie.com/embed/1dFvc42g8CA?rel=0&showinfo=0",
          "width": "560",
        },
        "ref": null,
        "rendered": null,
        "type": "iframe",
      },
      "type": "article",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <article
          className="video-frame"
>
          <iframe
                    allow="autoplay; encrypted-media"
                    allowFullScreen={true}
                    frameBorder="0"
                    height="315"
                    src="https://www.youtube-nocookie.com/embed/1dFvc42g8CA?rel=0&showinfo=0"
                    width="560"
          />
</article>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <iframe
            allow="autoplay; encrypted-media"
            allowFullScreen={true}
            frameBorder="0"
            height="315"
            src="https://www.youtube-nocookie.com/embed/1dFvc42g8CA?rel=0&showinfo=0"
            width="560"
/>,
          "className": "video-frame",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "allow": "autoplay; encrypted-media",
            "allowFullScreen": true,
            "frameBorder": "0",
            "height": "315",
            "src": "https://www.youtube-nocookie.com/embed/1dFvc42g8CA?rel=0&showinfo=0",
            "width": "560",
          },
          "ref": null,
          "rendered": null,
          "type": "iframe",
        },
        "type": "article",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
